README.md

```diff
@@ -166,6 +166,11 @@ When adding JavaDoc to code Oracle Java Platform extension assists by suggesting
 Oracle Java Platform extension provides Test Explorer view which allows to run all tests in a project, examine the results, go to source code and  run particular test.  
 ![Test Explorer](vscode/images/Test_explorer.png)
 
+### Localisation support
+Oracle Java Platform extension provides localisation support for Simplified chinese and japanese. 
+To switch language `Go to command Palette->Configure display language` select `zh-cn` for simplified chinese or `ja` for japanese
```

Comment:
Similar to the Changelog, do mention the language names in title case in alphabetic order:
*Japanese* and *Simplified Chinese*